src/jit/emitxarch.cpp

```diff
@@ -267,9 +268,12 @@ bool emitter::Is4ByteSSE4Instruction(instruction ins)
 bool emitter::TakesVexPrefix(instruction ins)
 {
     // special case vzeroupper as it requires 2-byte VEX prefix
-    if (ins == INS_vzeroupper)
+    switch (ins)
```

Comment:
this seems to be in a weird state. The sfence and prefetch instructions are missing.

Reply:
Yeah, rebasing to latest head which includes commit containing memory instructions - will fix